great_expectations/validator/validator.py

```diff
@@ -1711,9 +1726,11 @@ def validate(  # noqa: C901, PLR0912, PLR0913, PLR0915
         finally:
             self._active_validation = False
 
-        if getattr(data_context, "_usage_statistics_handler", None):
```

Comment:
@tyler-hoffman why are we changing this?
It's gone from 1 operation to 2.

Is it to help mypy type-narrow?

Reply:
Yeah, this was for type narrowing, but thinking about it again, the walrus operator is going to make this quite nice. I'll move that way.